centered_eight.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
<p>Not sure I would include these last cells, what do you think Ravin? Maybe a cell calling <code>to_netcdf</code>  and storing the result as <code>.nc</code>  file plus adding <code>.nc</code> files to gitignore? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>